recipes/retile.rb

```diff
@@ -38,8 +38,10 @@
   action  :nothing
   user    node[:valhalla][:user][:name]
   command "LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib pbfgraphbuilder -c \
-          #{node[:valhalla][:conf_dir]}/#{node[:valhalla][:config]} #{extracts}"
+          #{node[:valhalla][:conf_dir]}/#{node[:valhalla][:config]} #{extracts} \
+          >/tmp/pbfgraphbuilder.out 2>&1"
```

Comment:
we have the ability to log to file via config. i would prefer it that way if its ok with you.